main/gnome-remote-desktop/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "gnome-remote-desktop"
+pkgver = "47.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dvnc=false", "-Dsystemd=false", "-Dtests=false"]
```

Comment:
libvncserver exists so this should work i think

Reply:
I am trying to make rdp work properly for now.

Reply:
Added vncsever dep